app/models/card.rb

```diff
@@ -16,4 +18,15 @@ def check_duplication
   def set_default_date
     self.review_date = Date.today + 3.days
   end
+
+  def compare_translations(submitted_text)
+    origianl = self.translated_text.mb_chars.downcase.to_s
```

Comment:
Useless assignment to variable - `origianl`. Did you mean `original`?<br>Redundant `self` detected.